doc/install/manual-deployment.rst

```diff
@@ -196,11 +196,11 @@ The procedure is as follows:
 
 #. Populate the monitor daemon(s) with the monitor map and keyring. ::
 
-	sudo -u ceph ceph-mon [--cluster {cluster-name}] --mkfs -i {hostname} --monmap /tmp/monmap --keyring /tmp/ceph.mon.keyring
+	sudo -u ceph-mon [--cluster {cluster-name}] --mkfs -i {hostname} --monmap /tmp/monmap --keyring /tmp/ceph.mon.keyring
```

Comment:
quote from sudo(8)

-u user, --user=user
  Run the command as a user other than the default target user (usually root).  The user may be either a user name or a numeric user ID (UID) prefixed with the ‘#’ character (e.g., #0 for UID 0).  When running commands as a UID, many shells require that the ‘#’ be escaped with a backslash (‘\’).  Some security policies may restrict UIDs to those listed in the password database.  The sudoers policy allows UIDs that are not in the password database as long as the targetpw not set.  Other security policies may not support this.

Reply:
sorry，I don't quite understand what you mean.I will not get any response when I run this command as root.
ps.I noticed that you are in Shanghai.My English is not very good.So, if possible,can we talk in Chinese?  :)

Reply:
@liuenqi i was quoting the manpage of `sudo`. as Greg put, 

> it is invoking the ceph-mon binary as the "ceph" user

after your change, the command would be wrong.

Reply:
yes,I figured it out.I mistakenly think that "ceph" means command "ceph".I got it wrong.
Thank you for your reply.I wish the project is getting better and better.
：）